packages/opentelemetry-exporter-collector/src/platform/browser/CollectorExporterBrowserBase.ts

```diff
@@ -43,16 +49,20 @@ export abstract class CollectorExporterBrowserBase<
     super(config);
     this._useXHR =
       !!config.headers || typeof navigator.sendBeacon !== 'function';
+
     if (this._useXHR) {
-      this._headers = Object.assign(
-        {},
-        parseHeaders(config.headers),
-        baggageUtils.parseKeyPairsIntoRecord(
+      this._headers = {
```

Comment:
I would prefer to have 2 util functions
- prepareXHRHeaders / getXHRHeaders /  parseXHRHeaders etc.
- prepareBeaconHeaders ...

Currently the logic will be scattered between util and exporter, where exporter should not contain anymore logic then it is necessary and those functions will be straightforward and easy to understand. If someone want to change it you will change util class not the exporter.

Reply:
actually i think that the correct place would be to put the utility functions like the ones you mention here under 
https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-exporter-collector/src/util.ts

What do you think ? 
My motivation is because in this file there is already a helper method `parseheaders`. 
https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-exporter-collector/src/util.ts#L23

Reply:
Or a better one, what do you think ?

rename the file `util` to `send` under  https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-exporter-collector/src/platform/browser/util.ts
and export from it the functions `sendWithXhr`, `sendWithBeacon`. 

Then introduce a util.ts under `opentelemetry-js/blob/main/packages/opentelemetry-exporter-collector/src/platform/browser/` 
with the functions as helpers you mentioned above
- prepareXHRHeaders / getXHRHeaders / parseXHRHeaders etc.
-  prepareBeaconHeaders ...

Then send module and util module can be imported to `CollectorExporterBase` and collector exporter base can work with them.

Reply:
@obecny  a more consistent solution  is given by @MSNev  in context of blob and sendWithBeacon here: 
https://github.com/open-telemetry/opentelemetry-js/pull/2336#discussion_r681307342

So can i move the default header for xhr inside the send method of `CollectorExporterBase` ? 
i mean this logic under util.ts
https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-exporter-collector/src/platform/browser/util.ts#L59

to send method here: 
https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-exporter-collector/src/platform/browser/CollectorExporterBrowserBase.ts#L94

it should then look something like this:
```
if(this.xhr){
const headers = {...{"Content-Type":"application/json"},...this.headers}
sendWithXhr(body, this.url, headers, _onSuccess, _onError);
}
```
i would then close this PR and open a new one in order to be in context of xhr only , given the blob context PR will be handled here: https://github.com/open-telemetry/opentelemetry-js/pull/2336 

@obecny , @MSNev, what do you think for the above refactoring proposal for xhr ?